deployment/terraform/README.md

```diff
@@ -0,0 +1,48 @@
+# Description
+
+With this part of kuksa.cloud project it is possible to deploy a Keycloak server to Azure cloud platform secured with a 
+valid Let's Encrypt certificate without any manual steps. Please keep in mind that there is no claim to use this deployment
+in a productive setup.
+
+## Used Technologies  
+- Terraform
+- Azure
+- AKS
+- Helm
+- Let’s Encrypt
+- Keycloak
+
+# How-to use
+
+```sh
+# clone kuksa.cloud repository
+git clone https://github.com/eclipse/kuksa.cloud.git
+cd kuksa.cloud/deployment/terraform
+
+# login to Azure and set subscription
+az login
+az account set --subscription <ID>
+
+# prepare infrastructure configuration
+cp ./terraform.tfvars.template ./terraform.tfvars
+code ./terraform.tfvars
```

Comment:
this is misleading. why open vs code? needs a comment regarding what to change. if an editor is needed maybe switch to a more generic that is more likely already installed such as `vi`

Reply:
You're right, I will change this. I have no intention to give any editor recommendation.